tests/antithesis/fjall/singleton_driver_storage.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/bash
+export RUST_BACKTRACE=1
+
+if [[ -v ANTITHESIS_OUTPUT_DIR ]]; then
+  echo '{"antithesis_setup": { "status": "complete", "details": null }}' >> ${ANTITHESIS_OUTPUT_DIR}/sdk.jsonl
+fi
+
+exec /app/target/debug/rust-storage-bench run \
+  --compression none \
+  --backend fjall-nightly \
+  --data-dir .data \
+  --cache-size 536870912 \
```

Comment:
The cache size probably heavily depends on what machine the test is hosted on. ~50% of its RAM would make sense.